src/util-random.c

```diff
@@ -24,8 +24,8 @@
  * SEI CERT C Coding Standard MSC30-C
  */
 
-#include "suricata-common.h"
 #include "util-random.h"
+#include "suricata-common.h"
```

Comment:
why is this needed?

Reply:
It's not, not sure why I did it so I'm reverting it.